integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy

```diff
@@ -851,6 +851,31 @@ class UsersIT extends ITSupport implements CrudTestSupport {
         assertThat user.getCredentials().getProvider().getName(), equalTo(AuthenticationProviderType.FEDERATION.name())
     }
 
+    @Test
+    void testGetNextPageUrl() {
+        def user1 = create(client)
+        def user2 = create(client)
+        def user3 = create(client)
+        registerForCleanup(user1)
+        registerForCleanup(user2)
+        registerForCleanup(user3)
+
+        String pageSize = "2"
+
+        def userListFirstPage = client.http()
+            .addQueryParameter("limit", pageSize)
+            .get("/api/v1/users", UserList.class)
+        assertThat userListFirstPage, notNullValue()
+        assertThat userListFirstPage.getNextPageUrl(), notNullValue()
+        assertThat userListFirstPage.getProperties().get("currentPage").getProperties().get("items").collect().size(), is(2)
```

Comment:
What if the Org you're running this against doesn't have users list that spans 2 pages?

Reply:
I've created 3 users at lines 856-858.